chrome/content/zotero/xpcom/utilities.js

```diff
@@ -1527,14 +1484,14 @@ Zotero.Utilities = {
 	 */
 	"itemToCSLJSON":function(zoteroItem) {
 		if (zoteroItem instanceof Zotero.Item) {
-			zoteroItem = zoteroItem.toArray();
+			zoteroItem = Zotero.Utilities.Internal.itemToExportFormat(zoteroItem);
```

Comment:
Testing with the current beta, if a note or attachment item is argument, zoteroItem.creators is null, and itemToCSLJSON() explodes at [line 1539](https://github.com/aurimasv/zotero/blob/csl-json-export/chrome/content/zotero/xpcom/utilities.js#L1539). It may be intended that bare items never reach this function (?), but either wrapping the `for` block in a condition, or something like
``` javascript
if (!creators) {
    creators = [];
}
```
would avoid the error.